.github/workflows/tests.yml

```diff
@@ -18,6 +18,10 @@ jobs:
     concurrency:
       group: ${{ github.ref == 'refs/heads/main' && format('unit-tests-main-{0}', github.sha) || github.ref == 'refs/heads/develop' && format('unit-tests-develop-{0}', github.sha) || format('unit-tests-{0}', github.ref) }}
       cancel-in-progress: true
+    env:
```

Comment:
Is it possible to leave the `env` declaration at the original place? I think here we are providing env var to all the steps and this is not required.

Reply:
I don't know I follow an example from the doc and `env` variable where declared at the top. I will try to add them inside the step but not sure it can work.

Reply:
I guess the `env` instructions are handled first, so it should work.